python-models/models/MEPS-model-v1.ipynb

```
@@ -0,0 +1,945 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define schema for problematic columns\n",
+    "schema_overrides = {\n",
+    "    'HRWG53X': pl.Float64,  # Use float instead of int for decimal values\n",
+    "    # Add other problematic columns as needed\n",
+    "}\n",
+    "\n",
+    "df = pl.read_excel(\n",
+    "    \"/Users/orenj/Desktop/Projects/open-coverage/python-models/data/h243.xlsx\",\n",
+    "    schema_overrides=schema_overrides\n",
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)